code/datums/traits/good.dm

```diff
@@ -327,6 +327,43 @@
 	lose_text = "<span class='danger'>You forget how the periodic table works.</span>"
 	locked =  FALSE
 
+/datum/quirk/chemwhiz/add()
+	var/mob/living/carbon/human/H = quirk_holder
+	// I made the quirks add the same recipes as the trait books. Feel free to nerf this
+	H.mind.learned_recipes += list(
```

Comment:
Please turn this list and the others into global listw, that way they can be used for the other dozen or so other things that use literally the same list. We're trying to refactor these skills into global lists.